modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/views/table/dnd_table/Cell.tsx

```diff
@@ -53,6 +53,25 @@ const Cell = ({
 		if (columnName && heading && !isFixed && cellRef.current) {
 			const boundingClientRect = cellRef.current.getBoundingClientRect();
 
+			if (boundingClientRect.width < 70) {
+				const fontSize = getComputedStyle(cellRef?.current).fontSize;
+
+				const remSize = parseFloat(fontSize);
+
+				boundingClientRect.width += remSize * 4;
+			}
```

Comment:
In general, I like the idea of solution. We are extending size of small cells by an arbitrary amount, which may seem very opinionated, but we are already being opinionated by setting the width equal to column label width. If the column label is small in size, there is no reason why content would be small as well. If we ever remove the practice of setting width based on header, then this code would make no sense, but until then it's a reasonable improvement.

I believe there are few edge cases when this solution would harm UX:
1. If the table is close to fill full screen, this may push to earlier overflow. 
2. If all widths are small, it may cause an unnecessary overflow.

Still, I think in most cases this should be an improvement to UX, so :+1: